api/modules/food/constants.py

```diff
@@ -2,6 +2,9 @@
 This has used the Zomato Api
 For more information look into https://developers.zomato.com/api
 """
-ZOMATO_API = "9a656813579b698902619a05684d39c6"
+import os
+
+ORIGINAL_ZOMATO_API = "9a656813579b698902619a05684d39c6"
```

Comment:
Don't mention API Key here
Do local testing using the key, but do not commit the key in file